content/learn/05.communication/01.wire/wire.md

```diff
@@ -1,30 +1,92 @@
+- [What Is I2C?](#what-is-i2c)
+- [Qwiic & STEMMA QT](#qwiic--stemma-qt)
+- [Wire Library](#wire-library)
+- [Examples](#examples)
+
+## What Is I2C?
+The I2C protocol involves using two lines to send and receive data: a serial clock pin **(SCL)** that the Arduino Controller board pulses at a regular interval, and a serial data pin **(SDA)** over which data is sent between the two devices. 
+
+In I2C, there is one controller device, with one or more peripheral devices connected to the controllers SCL and SDA lines. 
+
+As the clock line changes from low to high (known as the rising edge of the clock pulse), a single bit of information is transferred from the board to the I2C device over the SDA line. As the clock line keeps pulsing, more and more bits are sent until a sequence of a 7 or 8 bit address, and a command or data is formed. When this information is sent - bit after bit -, the called upon device executes the request and transmits it's data back - if required - to the board over the same line using the clock signal still generated by the Controller on SCL as timing.
+
+Each device in the I2C bus is functionally independent from the controller, but will respond with information when prompted by the controller. 
 
 Because the I2C protocol allows for each enabled device to have it's own unique address, and as both controller and peripheral devices to take turns communicating over a single line, it is possible for your Arduino board to communicate (in turn) with many devices, or other boards, while using just two pins of your microcontroller.
 
-## Hardware & Software Needed
+An I2C message on a lower bit-level looks something like this:
+
+![An I2C Message](./assets/I2C.png)
+
+- The controller sends out instructions through the I2C bus on the data pin (SDA), and the instructions are prefaced with the address, so that only the correct device listens. 
+- Then there is a bit signifying whether the controller wants to read or write.
+- Every message needs to be acknowledged, to combat unexpected results, once the receiver has acknowledged the previous information it lets the controller know, so it can move on to the next set of bits.
+- 8 bits of data
+- Another acknowledgement bit
+- 8 bits of data
+- Another acknowledgement bit
+
+But how does the controller and peripherals know where the address, messages, and so on starts and ends? That's what the SCL wire is for. It synchronises the clock of the controller with the devices, ensuring that they all move to the next instruction at the same time.
+
+However, you are nearly never going to *actually* need to consider any of this, in the Arduino ecosystem we have the [Wire library](https://www.arduino.cc/reference/en/language/functions/communication/wire/) that handles everything for you.
+
+## Qwiic & STEMMA QT
```

Comment:
I think we need to rename this section to **Hardware Connection**, and list out all available options. For example, we don't list out the most basic option which is just two wires (SCL/SDA), and we don't list Groove as well. I think the structure could be simple:

```
## Hardware Connection
There are a number of way to connect I2C devices etc etc...
### Breakout Board (Pins)
circuit + how to connect to arduino

### Groove
circuit + how to connect to arduino

### Qwiic
circuit + board used

### Stemma
circuit + how to connect to arduino
```